api/features/tasks.py

```diff
@@ -1,17 +1,125 @@
+import json
+from datetime import timedelta
 from threading import Thread
+from typing import Optional, Union
 
-from environments.models import Webhook
-from features.models import FeatureState
+from django.db.models import Q
+from django.utils import timezone
+
+from environments.models import Environment, Webhook
+from features.models import (
+    Feature,
+    FeatureExport,
+    FeatureImport,
+    FeatureState,
+    FeatureStateValue,
+)
+from features.multivariate.models import MultivariateFeatureOption
+from features.value_types import BOOLEAN, INTEGER, STRING
+from projects.models import Project
+from task_processor.decorators import (
+    register_recurring_task,
+    register_task_handler,
+)
 from webhooks.constants import WEBHOOK_DATETIME_FORMAT
 from webhooks.webhooks import (
     WebhookEventType,
     call_environment_webhooks,
     call_organisation_webhooks,
 )
 
+from .constants import OVERWRITE, SKIP
 from .models import HistoricalFeatureState
 
 
+@register_recurring_task(
+    run_every=timedelta(hours=12),
+)
+def clear_stale_feature_imports_and_exports() -> None:
+    two_weeks_ago = timezone.now() - timedelta(days=14)
+    FeatureExport.objects.filter(created_date__lt=two_weeks_ago).delete()
+    FeatureImport.objects.filter(created_date__lt=two_weeks_ago).delete()
+
+
+@register_task_handler()
+def export_features_for_environment(
+    environment_id: int, tag_ids: Optional[list[int]] = None
+) -> None:
+    additional_filters = Q(
+        identity__isnull=True,
+        feature_segment__isnull=True,
+        feature_state_value__isnull=False,
+        feature__is_archived=False,
+    )
+
+    if tag_ids:
+        additional_filters &= Q(feature__tags__in=tag_ids)
+
+    feature_states = FeatureState.get_environment_flags_list(
+        environment_id=environment_id,
+        additional_filters=additional_filters,
+    )
```

Comment:
This is the primary method I relied upon to keep the feature states to export consistent with the rest of the application.